edb/lib/std/70-converters.edgeql

```diff
@@ -270,38 +270,77 @@ std::to_bytes(s: std::str) -> std::bytes {
 };
 
 
+CREATE SCALAR TYPE
+std::Endian EXTENDING enum<little, big>;
```

Comment:
Should it be `Little` and `Big`?

We *almost* always capitalize enum values. (But not, it seems, *always* always.)